modules/transport/http/src/org/apache/axis2/transport/http/AbstractHTTPSender.java

```diff
@@ -470,9 +470,24 @@ protected MessageFormatter populateCommonProperties(MessageContext msgContext, U
 
         httpMethod.setQueryString(url.getQuery());
 
-        httpMethod.setRequestHeader(HTTPConstants.HEADER_CONTENT_TYPE,
-                                    messageFormatter.getContentType(msgContext, format,
-                                                                    soapActionString));
+        // If adding the Content-Type header from the message formatter needs to be skipped
+        if ("true".equals(msgContext.getProperty(HTTPConstants.NO_DEFAULT_CONTENT_TYPE))) {
```

Comment:
can we modify the condition check like this 
`(Boolean.parseBoolean(msgContext.getProperty(HTTPConstants.NO_DEFAULT_CONTENT_TYPE)))`